tests/scripts/ci_helpers.sh

```diff
@@ -25,6 +25,45 @@ function install_deps() {
     date
 }
 
+function install_juju_simple() {
+    sudo snap install juju
+    mkdir -p ~/.local/share/juju
+    juju bootstrap localhost
+}
+
+function setup_juju_spaces() {
+    set -ex
+    date
+    juju add-model spacetest
+    juju add-space cluster
+    # Subnet value from LXD profile.
+    juju move-to-space cluster 10.85.4.0/24
+}
+
+function verify_juju_spaces_config() {
+    set -ex
+    date
+    local public="10.196.231.0/24"
+    local cluster="10.85.4.0/24"
+
+    # Verify the cluster network is correct subnet.
+    output=$(juju ssh microceph/0 -- "sudo microceph cluster config get cluster_network")
+    echo $output | grep $cluster
+
+    # Verify the public network is correct subnet.
+    output=$(juju ssh microceph/0 -- "sudo microceph cluster sql 'select value from config where key is \"public_network\"'")
+    echo $output | grep $public
```

Comment:
Similar to above

Reply:
Similar to above, we are fetching a single value against the "public_network" key (keys are unique constraint). Hence we would only get a single subnet in the output string.